pkg/apis/externaldns/types_test.go

```diff
@@ -267,6 +271,7 @@ func TestParseFlags(t *testing.T) {
 			title: "override everything via environment variables",
 			args:  []string{},
 			envVars: map[string]string{
+<<<<<<< HEAD
```

Comment:
seems wrong, I think this was due to a merge

Reply:
fixef